src/core_modules/capture-core-utils/date/date.utils.js

```diff
@@ -1,17 +1,25 @@
 // @flow
-import moment from '../moment/momentResolver';
+import { moment } from '../moment';
 
 export const displayTypes = {
     SHORT: 'short',
     MEDIUM: 'medium',
     LONG: 'long',
 };
 
-export function getMonthName(monthNumber: number) {
-    const date = moment([0, monthNumber - 1]);
-    return date.format('MMMM');
+/**
+ * Some locales are using numeral glyphs other than european. This method ensures the moment format method returns a value in european glyphs
+ * @param {*} momentDate: the moment instance
+ * @param {string} format: the moment format
+ * @returns {string} A formatted string with european glyphs
+ */
+export function getFormattedStringFromMomentUsingEuropeanGlyphs(momentDate: moment$Moment, format: string) {
```

Comment:
It looks like to me we could use some default values here. 

I can see for example that this function is used all over around our application now with format as `YYYY-MM-DD`, I would then set this value as default so that the user dont have to specify it each time. 
Also the default value for the `momentDate` could be `moment()`

In the end you will have something like this: 
```
export function getFormattedStringFromMomentUsingEuropeanGlyphs(momentDate: moment$Moment = moment(), format: string = 'YYYY-MM-DD') {
```
This will keep the signature of the function simpler for the end user and will also give them the ability to change both of those if they need. For example there are many times that we need to add instead of `moment()` a different instance then this would be easy to do but for the simple cases the user wont have to pass anything :)

Reply:
I'm a bit hesitant about this one. I can see your point and the benefits, thanks for pointing them out so we at least can make a conscious decision here. The disadvantage to adding defaults is that flow no longer will warn you that you're forgetting arguments if you intended to use something else than the defaults.

I'll meet you in the middle: No default for the momentDate, but I'm adding the default for the format. Hope that's ok. We can have a heated discussion later ;)